class/rental.rb

```diff
@@ -0,0 +1,28 @@
+require_relative 'book'
+require_relative 'person'
+
+require 'date'
+
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] All these attributes should have `attr_accessor`.

Reply:
added book variable, person variable to attr_accessor

Reply:
I didn't get what the reviewer meant.
the requirement for person class is:
Create a class Rental with the following:
@date instance variable, should be initialized in the constructor.
Setter and getter for @date (remember about attr_accessor).